lib/errors.js

```diff
@@ -287,11 +287,11 @@ function InvalidUpdateError(k, extra) {
 util.inherits(InvalidUpdateError, BucketsApiError);
 
 
-function InvalidParameterError(k, v) {
+function InvalidParameterError(v) {
     BucketsApiError.call(this, {
         restCode: 'InvalidParameter',
         statusCode: 400,
-        message: '"' + v + '" is invalid for "' + k + '"'
+        message: 'delimiter=' + v + ' is invalid '
```

Comment:
I wouldn't  hardcode *delimiter* in this function because the function is not specific to just the one delimiter use case. I would instead pass the string composed of `"delimiter " + delimiter into this function. Alternatively you could change this function to accept two parameters with the first being the parameter name and the second being the parameter value and compose the message from those two. I probably prefer the latter option because I think it's more clear.